core/src/com/cloud/storage/template/VhdProcessor.java

```diff
@@ -45,10 +49,12 @@
     private static final Logger s_logger = Logger.getLogger(VhdProcessor.class);
     StorageLayer _storage;
     private int vhdFooterSize = 512;
+    private int vhdCookieOffset = 8;
```

Comment:
@mike-tutkowski this was a fix that I had done for Swift to work with comrpessed templates. I've opened another PR which fixes this in Master. You can ignore these changes. I plan to remove them